test_suite/cloud/test_aws.py

```diff
@@ -302,6 +302,45 @@ def test_aws_instance_identity(self, host, instance_data, instance_data_aws):
             assert code in instance_data_aws['billingProducts'], \
                 'Expected billing code not found in instance document data'
 
+    def test_cmdline_nvme_io_timeout(self, host):
+        """
+        Check if default value of /sys/module/nvme_core/parameters/io_timeout is set to 4294967295.
+        BugZilla 1732506
+        """
+        expected_value = '4294967295'
+
+        if float(host.system_info.release) < 7.0:
+            pytest.skip('Not required in RHEL below 7.x')
+
+        with host.sudo():
+            assert host.file('/proc/cmdline').contains(f'nvme_core.io_timeout={expected_value}'), \
+                f'nvme_core.io_timeout should be set to {expected_value}'
+
+        if 'nvme' in host.check_output('lsblk'):
+            assert host.file('/sys/module/nvme_core/parameters/io_timeout').contains(expected_value), \
+                f'Actual value in io_timeout is not {expected_value}'
+
+    def test_udev_kernel(self, host):
+        """
+        /etc/udev/rules.d/80-net-name-slot.rules link to /dev/null
+        ks ref:
+        For cloud images, 'eth0' _is_ the predictable device name, since
+        we don't want to be tied to specific virtual (!) hardware
+        rm -f /etc/udev/rules.d/70*
+        ln -s /dev/null /etc/udev/rules.d/80-net-name-slot.rules
+        """
```

Comment:
I don't think this description helps. Maybe we could ask VirtQE to write a more clear onw